src/atoms/Text/__tests__/text.spec.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { expect } from 'chai';
+import { shallow, mount } from 'enzyme';
+
+import Text from 'atoms/Text';
+
+describe('<Text />', () => {
+
+  it('renders', () => {
+    const wrapper = shallow(<Text />);
+
+    expect(wrapper.length).to.equal(1);
```

Comment:
Please change this to the `expect(wrapper.type()).to.equal(Text);` that we'd discussed.